File: iniciante/1009/1009.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var nome = parseInt(lines.shift());
var sal_fixo = parseFloat(lines.shift());
var vendas = parseFloat(lines.shift());

var salario = sal_fixo + vendas * 0.15;

console.log(`TOTAL = R$ ${salario.toFixed(2)}`);

File: iniciante/1009/1009.dart
import 'dart:io';

// RESPOSTA DO BEECROWD (SEM NULL-SAFETY)
/*
void main() {
  String nome = stdin.readLineSync();
  double sal_fixo = double.parse(stdin.readLineSync());
  double vendas = double.parse(stdin.readLineSync());

  double salario = sal_fixo + vendas * 0.15;

  print("TOTAL = R\$ ${salario.toStringAsFixed(2)}");
}
*/

// RESPOSTA COM NULL-SAFETY
void main() {
  String nome = stdin.readLineSync()!;
  double sal_fixo = double.parse(stdin.readLineSync()!);
  double vendas = double.parse(stdin.readLineSync()!);

  double salario = sal_fixo + vendas * 0.15;

  print("TOTAL = R\$ ${salario.toStringAsFixed(2)}");
}
